src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVUtils.java

```diff
@@ -21,6 +21,8 @@
  */
 public final class SVUtils {
 
+    private static final String REFERENCE_GAP_INTERVAL_FILE_COMMENT_LINE_PROMPT = "##";
```

Comment:
Let's make this a single '#'.  User can still start lines with '##' if he or she prefers.
Otherwise, thumbs up!